### ISSUE TITLE
+ | Export-Csv $CSVFileName + ~ An empty pipe element is not allowed.

### ISSUE THREAD

**Issue by usman317319** [azure365pro/AzureAD_Inactive_Users#1]
I am not a professional and don't know why this error is occuring. Can you please guide me a little.

PS C:\Users\Muhammad Usman> .\AzureAD_inactive_users.ps1 -LastLogonDays 90 -CSVFileName 90daysinactive.csv
At C:\Users\Muhammad Usman\AzureAD_inactive_users.ps1:30 char:1
+ | Export-Csv $CSVFileName
+ ~
An empty pipe element is not allowed.
    + CategoryInfo          : ParserError: (:) [], ParseException
    + FullyQualifiedErrorId : EmptyPipeElement

**Comment by guernseyb** [comment]
Found this to work after moving bar to end of 56th line. 57th line should just read Export-Csv $CSVFileName

**Comment by usman317319** [comment]
Yes sir that issue is resolved the script is now working without error. Thank you soo much.....
But also now I am getting this.

cmdlet Export-Csv at command pipeline position 1
Supply values for the following parameters:
InputObject:
